modules/trusted-firmware-m/Kconfig

```diff
@@ -0,0 +1,140 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+config TFM_MINIMAL
+	bool "Use minimal TFM build"
+	imply TFM_IPC
+
+if !TFM_MINIMAL
+source "$(ZEPHYR_BASE)/modules/trusted-firmware-m/Kconfig"
```

Comment:
Maybe a comment on why we do this and what we gain be using this principle would be nice.
Either here, or in the beginning of the file.

Reply:
Agree